resources/views/flights/index.blade.php

```diff
@@ -0,0 +1,222 @@
+            .catch(function (err) {
+                const errorMessage = document.getElementById('flight_form_error');
+                if (err.response?.status === 422) {
+                    if (err.response?.data?.error?.fields) {
+                        errorMessage.textContent = Object.values(err.response.data.error.fields);
+                    }
+                }
+            });
+        }
+
+        function loadFlights(page = 1) {
+            currentPage = page;
+            axios.get(`/api/flights?page=${currentPage}`).then(res => {
+                const tbody = document.getElementById('flight_table_body');
+                const template = document.getElementById('flight_row_template');
+
+                const rows = tbody.querySelectorAll('tr');
+                for (const row of rows) {
+                    if (row.id !== 'flight_row_template') {
+                        row.remove();
+                    }
+                }
+                res.data.data.forEach(flight => {
+                    const row = template.cloneNode(true);
+                    row.id = '';
+                    row.classList.remove('hidden');
+
+                    row.querySelector('.id-cell').textContent = flight.id;
+                    row.querySelector('.origin-cell').textContent = flight.departure_city_name;
+                    row.querySelector('.destination-cell').textContent = flight.arrival_city_name;
+                    row.querySelector('.airline-cell').textContent = flight.airline_name;
+                    row.querySelector('.departure-cell').textContent = formatDate(flight.departure_time);
+                    row.querySelector('.arrival-cell').textContent = formatDate(flight.arrival_time);
+
+                    const deleteBtn = row.querySelector('.delete-btn');
+                    deleteBtn.addEventListener('click', () => deleteFlight(flight.id));
+
+                    const editBtn = row.querySelector('.edit-btn');
+                    editBtn.addEventListener('click', () => {
+                        window.location.href = `/flights/${flight.id}/edit`;
+                    });
+
+                    tbody.appendChild(row);
+                });
+                renderPagination(res.data.meta);
+            });
+        }
+
+        function deleteFlight(id) {
+            if (!confirm('Are you sure you want to delete this flight?')) return;
+            axios.delete(`/api/flights/${id}`).then(() => {
+                loadFlights(currentPage);
+            }).catch(error => {
+                document.getElementById('flight_form_error').textContent = error.message;
+            });
+        }
+
+        function formatDate(datetime) {
+            const date = new Date(datetime);
+
+            const year = date.getFullYear();
+            const month = date.getMonth() + 1;
+            const day = date.getDate();
+
+            return (year + '-' + (month < 10 ? '0' + month : month) + '-' + (day < 10 ? '0' + day : day));
```

Comment:
Try to use a js function for format (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)